bin/utils/subkey/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "subkey"
-version = "2.0.2"
+version = "3.0.0"
```

Comment:
I think we need this? As it has a very minor breaking change to the output. WDYT?